projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+{
+  "itemGroup.slashblade": "拔刀剑",
+  "itemGroup.slashblade.desc": "堕入阿修罗之道",
+
+  "attribute.name.generic.reachDistance":"触及距离",
+
+  "stat.slashblade.sword_summoned": "召唤幻影剑",
+  "entity.slashblade.abstract_summoned_sword": "幻影剑",
+
+  "slashblade.tooltip.killcount": "杀敌数：%1$s",
+  "slashblade.tooltip.refine": "强化等级：%1$s",
+
+  "slashblade.tooltip.material": "合成条件",
+  "slashblade.tooltip.material.requiredobjects.anvil": "合成场所：铁砧",
+  "slashblade.tooltip.material.killcount": "杀敌数 ≧%1$s",

```

```suggestion
  "slashblade.tooltip.material.killcount": "杀敌数≧%1$s",
```
标点与汉字之间不需要加空格
此类问题请全文通查